src/resolvers/resolvers.js

```diff
@@ -0,0 +1,76 @@
+import { GraphQLScalarType, Kind } from "graphql";
+import { BookService } from "../services/bookService.js";
+import { AuthorService } from "../services/authorService.js";
+
+export const resolvers = {
+    Date: new GraphQLScalarType({
```

Comment:
also there are libraries which provider with other scalar, see if you can find them and see how they works.